dist/plugin.json

```diff
@@ -6,7 +6,7 @@
   "name": "Scalyr",
   "id": "scalyr-datasource",
   "info": {
-    "version": "2.0.0",
+    "version": "2.1.0",
```

Comment:
According to the `CHANGELOG.md` this should be version `2.2.0`, with `2.1.0` `Support for annotation queries`.
Also, need to update `CHANGELOG.md`.